### PR TITLE
SigningService: Add default value for `initial_reconnect_delay` setting

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/692

### CODE REVIEW

File: signing_service/src/tests/tests.py [hunk 1]
```diff
@@ -113,23 +114,24 @@ def setUp(self):
...
+        sys.argv += ['127.0.0.1', '--initial_reconnect_delay', '1', '--concent-cluster-port', '8000']
 
         args = _parse_arguments()
 
         self.assertEqual(args.concent_cluster_host, '127.0.0.1')
         self.assertEqual(args.concent_cluster_port, 8000)
 
-    def test_that_argument_parser_should_parse_correct_input_and_use_default_port(self):
-        sys.argv += ['127.0.0.1', '1']
+    def test_that_argument_parser_should_parse_correct_input_and_use_default_values(self):
```
Comment (dybi) [inline signing_service/src/tests/tests.py]: something's wrong here 
1. test is called: `test_that_argument_parser_should_parse_correct_input_and_use_default_values`
2. some value of `initial_reconnect_delay` is being passed (`1`)
3. there's an assert that checks if `initial_reconnect_delay` is equal to `SIGNING_SERVICE_DEFAULT_INITIAL_RECONNECT_DELAY`
So basically, in this tests we check if user has entered default: `SIGNING_SERVICE_DEFAULT_INITIAL_RECONNECT_DELAY` ;)

File: signing_service/src/tests/tests.py [hunk 2]
```diff
@@ -113,23 +114,24 @@ def setUp(self):
         sys.argv = sys.argv[:1]
 
     def test_that_argument_parser_should_parse_correct_input(self):
-        sys.argv += ['127.0.0.1', '1', '--concent-cluster-port', '8000']
+        sys.argv += ['127.0.0.1', '--initial_reconnect_delay', '1', '--concent-cluster-port', '8000']
```
Comment (dybi) [inline signing_service/src/tests/tests.py]: there should be check for `initial_reconnect_delay` as well

File: signing_service/src/tests/tests.py (hunk 1 above)
Comment (rwrzesien) [reply signing_service/src/tests/tests.py]: :+1: done

File: signing_service/src/tests/tests.py (hunk 2 above)
Comment (rwrzesien) [reply signing_service/src/tests/tests.py]: :+1: done

**Pull request merged**